internal/helm/helmrepoadapter/gorm_repo_store.go

```diff
@@ -0,0 +1,137 @@
+// Copyright © 2020 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helmrepoadapter
+
+import (
+	"context"
+
+	"emperror.dev/errors"
+	"github.com/jinzhu/gorm"
+
+	"github.com/banzaicloud/pipeline/internal/common"
+	"github.com/banzaicloud/pipeline/internal/helm"
+)
+
+// repositoryModel describes the common cluster model.
+type repositoryModel struct {
+	gorm.Model
+
+	OrganizationID   uint   `gorm:"unique_index:unique_per_org"`
```

Comment:
```suggestion
	OrganizationID   uint   `gorm:"unique_index:idx_org_name"`
```

or something like that